lib/Field/stories/FieldStory.js

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { action } from '@storybook/addon-actions';
+import { Field } from 'lib';
+import { boolean, text } from '@storybook/addon-knobs';
+import StoryItem from '../../../stories/styleguide/StoryItem';
+
+const mockValidations = [
+  {
+    text: 'Validation failed text',
+    hasFailed: true
+  },
+  {
+    text: 'Validation text',
+    hasFailed: false
+  }
+];
+
+const mockActions = [
+  {
+    text: 'test action',
+    clickHandler: action('action clicked')
+  },
+  {
+    text: 'test action 2',
+    clickHandler: action('action 2 clicked')
+  }
+];
+
+storiesOf('Components', module).add('Field', () => {
+  const dir = text('Text direction', 'ltr');
```

Comment:
It could be cool if we used radios here instead of letting the user type in the dir.

https://www.npmjs.com/package/@storybook/addon-knobs#radio-buttons

Reply:
done 👍